src/server/config/json/auth.callback.js

```diff
@@ -29,4 +29,8 @@ export default {
       msg: 'Authentication failed. Please sign in first.',
     },
   },
+  fail: {
```

Comment:
How does this fail differ from all the other fails?? 
At least provide a comment